editor/plugins/org.fusesource.ide.launcher/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,7 @@ Require-Bundle: org.eclipse.jdt.launching,
  org.eclipse.ui.ide,
  org.eclipse.emf.transaction;bundle-version="1.9.0",
  org.eclipse.jdt.core,
- org.jboss.tools.jmx.core;bundle-version="1.8.3",
+ org.jboss.tools.jmx.core,
```

Comment:
better to keep the minimal version otherwise we will have issues with remote debug